dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/Api/V1/GetAllUsersTests.cs

```diff
@@ -1,13 +1,27 @@
+using System.Net;
 using TeacherIdentity.AuthServer.Api.V1.Responses;
 using TeacherIdentity.AuthServer.Oidc;
 
 namespace TeacherIdentity.AuthServer.Tests.EndpointTests.Api.V1;
 
+[Collection(nameof(DisableParallelization))]
 public class GetAllUsersTests : TestBase
 {
     public GetAllUsersTests(HostFixture hostFixture)
         : base(hostFixture)
     {
+        Task.Run(ClearNonTestUsers).Wait();
```

Comment:
Implement `IAsyncLifetime` and call `ClearNonTestUsers()` from `InitializeAsync`